src/notation/view/internal/dynamicpopupmodel.h

```diff
@@ -0,0 +1,40 @@
+#ifndef MU_NOTATION_DYNAMICPOPUPMODEL_H
+#define MU_NOTATION_DYNAMICPOPUPMODEL_H
+
+#include <engraving/iengravingfontsprovider.h>
+#include <engraving/style/style.h>
+#include <view/abstractelementpopupmodel.h>
+
+#include <QList>
+#include <QObject>
+#include <QVariantList>
```

Comment:
Code style thingy:
```suggestion
#include <QList>
#include <QObject>
#include <QVariantList>

#include "engraving/iengravingfontsprovider.h"
#include "engraving/style/style.h"

#include "view/abstractelementpopupmodel.h"
```
(We often prefer to include standard library headers first, then Qt headers, then headers from other modules, and then headers from the current module. The big exception to that, is in cpp files, where the corresponding header file is included before all other headers. 
Also, for our "own" headers we prefer the `""` syntax over the `<>` syntax.)